osu.Game.Tests/Visual/Gameplay/TestScenePlayerLoader.cs

```diff
@@ -97,6 +118,61 @@ public void TestModReinstantiation()
             AddAssert("player mods applied", () => playerMod2.Applied);
         }
 
+        [Test]
+        public void TestMutedNotification()
+        {
+            AddStep("set master volume to 0%", () => audioManager.Volume.Value = 0);
+            AddStep("reset notification lock", () => PlayerLoader.ResetNotificationLock());
+            //AddStep("reset notification overlay", () => notificationOverlay);
```

Comment:
can be removed

Reply:
Can't be removed, other tests run before TestMutedNotification() and might change state of the game like the notification lock by PlayerLoader, and the master volume might be 100% for AppVeyor and other new clients.

Reply:
Why is this comment here? (it's not even executable)

Reply:
https://github.com/ppy/osu/pull/6126#issuecomment-533142187
Like I said this was a kinda an unintended commit without PR quality control.